LobbyServer2/LobbyServer/LobbyServerProtocol.cs

```diff
@@ -202,6 +202,12 @@ private void HandleUpdateRemoteCharacterRequest(UpdateRemoteCharacterRequest req
                 return;
             }
 
+            if (MatchmakingManager.IsQueued(GroupManager.GetPlayerGroup(AccountId)))
+            {
+                // Not allowed to change characters while in queue.
+                return;
+            }
```

Comment:
Is there a problem here? UI suggest that you can switch characters while in queue.

And in any case, we need to send a response.

Reply:
Meh guess we cant remove it

Reply:
And an error message if it doesn't show the one in response